src/main/java/org/broadinstitute/hellbender/tools/gvs/common/CostObservability.java

```diff
@@ -0,0 +1,122 @@
+
+public class CostObservability {
+    static final Logger logger = LogManager.getLogger(org.broadinstitute.hellbender.tools.gvs.common.CostObservability.class);
+
+    private final TableName costObservabilityTable;
+
+    public CostObservability(String projectID, String datasetName, String costObservabilityTableName) {
+        this.costObservabilityTable = TableName.of(projectID, datasetName, costObservabilityTableName);
+    }
+
+    public TableSchema getCostObservabilityTableSchema() {
+        TableSchema.Builder builder = TableSchema.newBuilder();
+        builder.addFields(
+                TableFieldSchema.newBuilder().setName("call_set_identifier").setType(TableFieldSchema.Type.STRING).setMode(TableFieldSchema.Mode.REQUIRED).build()
+        );
+        builder.addFields(
+                TableFieldSchema.newBuilder().setName("step").setType(TableFieldSchema.Type.STRING).setMode(TableFieldSchema.Mode.REQUIRED).build()
+        );
+        builder.addFields(
+                TableFieldSchema.newBuilder().setName("call").setType(TableFieldSchema.Type.STRING).setMode(TableFieldSchema.Mode.NULLABLE).build()
+        );
+        builder.addFields(
+                TableFieldSchema.newBuilder().setName("shard_identifier").setType(TableFieldSchema.Type.STRING).setMode(TableFieldSchema.Mode.NULLABLE).build()
+        );
+        builder.addFields(
+                TableFieldSchema.newBuilder().setName("call_start_timestamp").setType(TableFieldSchema.Type.TIMESTAMP).setMode(TableFieldSchema.Mode.REQUIRED).build()
+        );
+        builder.addFields(
+                TableFieldSchema.newBuilder().setName("event_timestamp").setType(TableFieldSchema.Type.TIMESTAMP).setMode(TableFieldSchema.Mode.REQUIRED).build()
+        );
+        builder.addFields(
+                TableFieldSchema.newBuilder().setName("event_key").setType(TableFieldSchema.Type.STRING).setMode(TableFieldSchema.Mode.REQUIRED).build()
+        );
+        builder.addFields(
+                TableFieldSchema.newBuilder().setName("event_bytes").setType(TableFieldSchema.Type.INT64).setMode(TableFieldSchema.Mode.REQUIRED).build()
+        );
+        return builder.build();
+    }
+
+    public void writeCostObservability(String callSetIdentifier, String step, String call, String shardIdentifier,
+                                       Date callStartTimestamp, Date eventTimestamp, String eventKey, long eventBytes) {
+        final ExponentialBackOff backoff = new ExponentialBackOff.Builder().
+                setInitialIntervalMillis(2000).
+                setMaxIntervalMillis(30000).
+                setMultiplier(2).
+                setRandomizationFactor(0.5).
+                build();
+        int retryCount = 0;
+        int maxRetries = 3;
+
+        while (true) {
+            // This uses the _default stream since it (a) commits immediately and (b) doesn't count
```

Comment:
what does the single _ mean?

Reply:
Really not sure - this was code that was largely refactored from 'LoadStatus'.  I will remove it.